image_classification/index.html

```diff
@@ -150,6 +150,11 @@
   <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
     integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
     crossorigin="anonymous"></script>
+  <script>
+    if ( typeof module === "object" && typeof module.exports === "function") {
+      window.jQuery = window.$ = require('jquery');
```

Comment:
Same here, add annotation here to explain why we need this.

Reply:
Thanks, updated yet, PTAL again! @Honry

Reply:
I met a problem that it failed to `require('jquery')` on windows: `Cannot find module 'jquery'`, while it's ok on linux. Any suggestions? @huningxin Thanks!